java/fury-core/src/main/java/org/apache/fury/meta/ClassDef.java

```diff
@@ -237,8 +237,14 @@ public List<Descriptor> getDescriptors(ClassResolver resolver, Class<?> cls) {
         Descriptor newDesc = fieldInfo.toDescriptor(resolver);
         if (descriptor != null) {
           // Make DescriptorGrouper have consistent order whether field exist or not
-          descriptor = descriptor.copyWithTypeName(newDesc.getTypeName());
-          descriptors.add(descriptor);
+          // fury builtin types skip
+          if (newDesc.getRawType().getTypeName().contains("apache.fury")
```

Comment:
`newDesc.getRawType().getTypeName().contains("apache.fury")` looks too hack, this should be removed